features/buyer/requirements.feature

```diff
@@ -196,3 +196,33 @@ Scenario: There is no 'Publish requirements' button for an incomplete requiremen
   And I have created an individual specialist requirement
   When I click 'Review and publish your requirements'
   Then I don't see the 'Publish requirements' button
+
+
+Scenario: Ensure we can log in as a buyer with a closed brief
+  Given I am logged in as the buyer of a closed brief with responses
+  When I click the 'View your account' link
+  Then I see that the 'Closed requirements' summary table has 1 or more entries
+
+
+@requirements @award
+Scenario: Award a requirement to a winning supplier
+  Given I am logged in as the buyer of a closed brief with responses
```

Comment:
If you put the previous scenario as a background, you shouldn't need this line :crossed_fingers: